mobile/docs/conf.py

```diff
@@ -103,13 +103,12 @@ def setup(app):
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 
-if not os.environ.get('ENVOY_DOCS_VERSION_STRING'):
+# The short X.Y version.
+if not (version := os.environ.get("ENVOY_DOCS_VERSION_STRING")):
     raise Exception("ENVOY_DOCS_VERSION_STRING env var must be defined")
 
-# The short X.Y version.
-version = os.environ['ENVOY_DOCS_VERSION_STRING']
 # The full version, including alpha/beta/rc tags.
-release = os.environ['ENVOY_DOCS_VERSION_STRING']
+release = version
```

Comment:
Seems there's a mismatch between the description (comment) of `release` and `version` described 4 lines above, though both should be the same.

Reply:
there are _too many vars_ here - in this PR im doing the minimum to get things working but happy to remove some of this superflous code in a follow up

fwiw - in a follow up - this will no longer happen here anyway i think - at least in relation to publishing (rather than testing)

Reply:
SGTM.
Next time I suggest adding a TODO, but no need for it in this PR.